### PR TITLE
Support Flysystem v3

### PR DESCRIPTION
**Pull request by jmsche**

### CODE REVIEW

**Comment by freekmurze** [comment]
Thanks!

File: src/DropboxAdapter.php [hunk 1]
```diff
@@ -60,6 +62,11 @@ public function fileExists(string $path): bool
         }
     }
 
+    public function directoryExists(string $path): bool
+    {
+        return $this->fileExists($path);
```
Comment (GrahamCampbell) [inline src/DropboxAdapter.php:67]: This looks incorrect?

Comment (jmsche) [reply src/DropboxAdapter.php:67]: According to the Dropbox API it seems correct: https://www.dropbox.com/developers/documentation/http/documentation#files-get_metadata

Also I upgraded to latest version earlier & added a check to see if a directory exists or not before creating it, it seems to be working great.

Comment (GrahamCampbell) [reply src/DropboxAdapter.php:67]: directoryExists should not return true if a file exists at that path, only if a directory exists.

**Pull request merged**